exercises/practice/series/series_test.go

```diff
@@ -1,32 +1,3 @@
-// Define two functions:  (Two? Yes, sometimes we ask more out of Go.)
```

Comment:
We should probably move some parts of this to an `instructions.append.md` file.

Reply:
I forgot to mention that this exercise already has an [instructions append](https://github.com/exercism/go/blob/main/exercises/practice/series/.docs/instructions.append.md) with the same content as this comment. Sorry for the trouble